app/javascript/hooks/useLanguageStringInUrl.tsx

```diff
@@ -0,0 +1,7 @@
+import { getRoutePrefix } from "../util/languageUtil"
+
+const useLanguageStringInUrl = () => {
+  return window.location?.pathname && getRoutePrefix(window.location.pathname)
```

Comment:
can we use `getCurrentLanguage` in `languageUtil`?

Reply:
Didn't see that function. GetCurrentLanguage feels better suited. updated!